[FSTORE-1789] Data source docs (#498)
logicalclocks/logicalclocks.github.io: docs/user_guides/fs/feature_group/create_external.md

## Create using the UI

You can also create a new feature group through the UI. For this, navigate to the `Data Source` section and select existing credentials or create new ones for your prefered data source. 

<p align="center">
  <figure>
    <img src="../../../../assets/images/guides/feature_group/data_source.png" style="border: 10px solid #f5f5f5" alt="Data Source UI">
  </figure>
</p>

> If you have existing credentials, simply proceed by clicking `Next: Select Tables`. If you not, create and save the credentials first. 

Reviewer: Maybe it also makes sense to provide links for different DS docs here, as a guide on how exactly to fill in new credentials.

<p align="center">
  <figure>
    <img src="../../../../assets/images/guides/feature_group/credentials_selection.png" alt="setup credentials in Data Sources">
  </figure>
</p>

> The database navigation structure depends on your specific data source. You'll navigate through the appropriate hierarchy for your platform—such as Database → Schema → Table for Snowflake, or Project → Dataset → Table for BigQuery. In the UI you can select one or more tables, for each selected table, you must designate one or more columns as primary keys before proceeding. You can also optionally a single column as a timestamp for the row (supported types are timestamp, date and bigint), edit names and data types of individual columns you want to include.

Reviewer: It is nice to provide links here for the examples (link to Snowflake guide for Snowflake, etc.).

Reviewer: > You can also optionally a single column

should be
```
You can also optionally select/designate a single column
```
Otherall, the sentence seems to be overloaded. It is better to split it and probably move all of these "requirements and options" to a separate paragraph, as it doesn't really fall under the topic sentence

> The database navigation structure depends on your specific data source.

<p align="center">
  <figure>
    <img src="../../../../assets/images/guides/feature_group/ext_table_selection.png" style="border: 10px solid #f5f5f5" alt="Select Table in Data Sources for External feature Group">
  </figure>
</p>

<p align="center">
  <figure>
    <img src="../../../../assets/images/guides/feature_group/primary_key_selection.png" style="border: 10px solid #f5f5f5" alt="select details of external feature group">
  </figure>
</p>

> Complete the creation by clicking `Next: Review Configuration` at the bottom of the page, you will be prompted with a final validation window where you will be able to create a name for your external feature group. 

Reviewer: ```suggestion
Complete the creation by clicking `Next: Review Configuration` at the bottom of the page. As the last step, you will name the external feature groups and confirm their creation.
```
